openslides/assignment/serializers.py

```diff
@@ -136,5 +135,5 @@ class Meta:
             'status',
             'assignmentcandidate_set',
```

Comment:
Done.

Is it ok that all users with the permission to see assignments can see which user is blocked?